releasey/README.md

```diff
@@ -0,0 +1,131 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+
+# Polaris Release Automation Scripts
+
+This directory contains scripts to automate the Apache Polaris release process, following the [official release guide](https://github.com/apache/polaris/blob/main/docs/release-guide.md).
+
+## Release Scripts
+
+These scripts are used to perform actual releases:
+
+### 1. Prerequisites Verification
+```bash
+./01-prerequisites.sh
+```
+Verifies that your environment is ready for releases:
+- GPG setup and key configuration
+- Maven/Gradle credentials
+- Git remote setup for Apache repository
+- GitHub token for test status verification
+
+### 2. Create Release Branch
+```bash
+./02-create-release-branch.sh --version 1.0.0-incubating [--commit HEAD] [--recreate]
```

Comment:
Nit: I think the version string needs `rcN`, it will error out otherwise.
```
./02-create-release-branch.sh --version 1.0.0-incubating-rc1
```

Reply:
Good point, fixing...